src/editor-mathfield/mode-editor-math.ts

```diff
@@ -193,6 +196,38 @@ export class MathModeEditor extends ModeEditor {
     );
     if (!newAtoms) return false;
 
+    const newPlaceholders = findPlaceholders(newAtoms);
+    newPlaceholders.forEach((placeholder) => {
+      if (
+        placeholder.placeholderId &&
+        !model.mathfield._placeholders.has(placeholder.placeholderId)
+      ) {
+        const element = new MathfieldElement();
+        element.classList.add('mathfield');
+        element.style.display = 'inline-block';
+        element.style.zIndex = '1001';
+        element.style.fontSize = '60%';
```

Comment:
Why do you need to change the fontSize?

Reply:
I needed to scale down the size of the font because the mathfield was bigger than required.